spec/controllers/queue_items_controller_spec.rb

```diff
@@ -168,12 +168,12 @@
       end
 
       it "sets the flash danger message" do
-        post :update_queue, queue_items: [{id: light_saber.id, position: 3.4}, {id: r2d2.id, position: 1}]
+        post :update_queue, queue_items: [{id: light_saber.id, position: 3}, {id: r2d2.id, position: 2.1}]
```

Comment:
Line is too long. [106/80]<br>Space inside { missing.<br>Space inside } missing.